src/tools/perf/lib/ucp_tests.cc

```diff
@@ -6,6 +6,10 @@
 * See file LICENSE for terms.
 */
 
+#ifndef __STDC_LIMIT_MACROS
+#  define __STDC_LIMIT_MACROS /* for UINT**_MAX values */
```

Comment:
can we remove this and use UCS_MASK_SAFE(64) or std::numeric_limits<ucp_tag_t>::max() for example instead of UINT64_MAX?

Reply:
removed all __STDC_LIMIT_MACROS from gtests